crashsimilarity/models/similarity/doc2vec_similarity.py

```diff
@@ -0,0 +1,49 @@
+from gensim.models.doc2vec import TaggedDocument
+
+from crashsimilarity.models.similarity.base import Algorithm, GenericSimilarity
+
+
+class Doc2VecSimilarity(Algorithm):
+    def __init__(self, wmd_calculator):
+        """
+        :type wmd_calculator: WMDCalculator
+        """
+        self._calculator = wmd_calculator
+
+    def top_similar_traces(self, stack_trace, corpus, top_n=10):
+        _stack_trace = stack_trace.words if isinstance(stack_trace, TaggedDocument) else stack_trace
+        _corpus = [c.words if isinstance(c, TaggedDocument) else c for c in corpus]
+
+        distances = list(enumerate([self._calculator.wmdistance(_stack_trace, c) for c in _corpus]))
+        return sorted(distances, key=lambda x: x[1])[:10]
+        # distances = []
```

Comment:
It is usually bad practice to commit commented code into master, but I have a reason to do it.
Now, i sometimes have some not very accurate results after floats comparison. (like 9 of 10 top signatures are correct) While in general the commented code is correct. I have a test for this method (which is failed because of floats), so this code will be uncommented as soon as i will figure out what to do with floats.